service/src/main/java/greencity/service/CommentServiceImpl.java

```diff
@@ -3,36 +3,26 @@
 import greencity.achievement.AchievementCalculation;
 import greencity.constant.ErrorMessage;
 import greencity.dto.PageableDto;
-import greencity.dto.comment.AddCommentDtoRequest;
-import greencity.dto.comment.AddCommentDtoResponse;
-import greencity.dto.comment.CommentAuthorDto;
-import greencity.dto.comment.CommentDto;
+import greencity.dto.comment.*;
```

Comment:
do not use wildcard imports

Reply:
@K4sik 

In intellij you could go to settings -> editor -> code style -> java

make sure Use single class import is enabled
in class count to use import with '*' and names count to use static import with '*' set some big number (e.g 999) and intellij idea would import each class separately.